packages/nextlove/src/types/index.ts

```diff
@@ -64,10 +64,12 @@ export type QueryArrayFormats = readonly QueryArrayFormat[]
 
 export interface SetupParams<
   AuthMW extends AuthMiddlewares = AuthMiddlewares,
-  GlobalMW extends Middleware<any, any>[] = any[]
+  GlobalMW extends Middleware<any, any>[] = any[],
+  GlobalAfterAuthMW extends Middleware<any, any>[] = any[]
 > {
   authMiddlewareMap: AuthMW
   globalMiddlewares: GlobalMW
+  globalAfterAuthMiddlewares?: GlobalAfterAuthMW
```

Comment:
🐧 
```suggestion
  globalMiddlewaresAfterAuth?: GlobalAfterAuthMW
```

is slightly less yoda-y